administrative-sdk/pronunciationChallenge.js

```diff
@@ -53,60 +53,22 @@ class PronunciationChallenge {
   }
 
   /**
-   * Callback used by createPronunciationChallenge.
-   *
-   * @callback Sdk~pronunciationChallengeCreatedCallback
-   * @param {its.PronunciationChallenge} challenge Updated speech challenge domain model instance.
-   */
-  pronunciationChallengeCreatedCallback(challenge) {}
-
-  /**
-   * Error callback used by createPronunciationChallenge.
-   *
-   * @callback Sdk~pronunciationChallengeCreatedErrorCallback
-   * @param {object[]} errors Array of errors.
-   * @param {its.PronunciationChallenge} challenge Pronunciation challenge domain model instance with unapplied changes.
-   */
-  pronunciationChallengeCreatedErrorCallback(errors, challenge) {}
-
-  /**
    * Create a pronunciation challenge.
    *
    * @param {its.PronunciationChallenge} challenge A pronunciation challenge object.
    * @param {Sdk~pronunciationChallengeCreatedCallback} [cb] The callback that handles the response.
    * @param {Sdk~pronunciationChallengeCreatedErrorCallback} [ecb] The callback that handles the error response.
```

Comment:
Missing documentation update. Callbacks are no longer taken and a promise is returned.